htdocs/api/class/api_setup.class.php

```diff
@@ -1220,7 +1220,8 @@ public function getPaymentTerms($sortfield = "sortorder", $sortorder = 'ASC', $l
 	}
 
 	/**
-	 * Get the list of shipping methods.
+	 * Get the list of shipping methods. This operation is deprecated, use
+	 * /setup/dictionary/shipment_methods instead.
```

Comment:
This url suggested seems the same than this one. ?

Reply:
![image](https://user-images.githubusercontent.com/3624836/107232252-7f787d80-6a21-11eb-8543-4036d684b844.png)